packages/big-design/src/components/Counter/styled.ts

```diff
@@ -0,0 +1,34 @@
+import { remCalc, theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+import { StyleableButton } from '../Button/private';
+import { StyledInput, StyledInputWrapper, StyledInputWrapperProps } from '../Input/private';
+
+export const StyledCounterButton = styled(StyleableButton)`
```

Comment:
There may be some other things you need to override here (`height`, `width`, `padding`, etc.). I would also check to see what this looks like on mobile.

Reply:
I don't see any obvious reasons to override most of the styles, however you're right that I needed to change the width on mobile. Fixed.